src/v2/cookbook/form-validation.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Form Validation
+type: cookbook
+order: 1.2
+---
+
+## Simple Example
+
+Form validation is natively supported by the browser, but sometimes different browsers will handle things in a manner which makes relying on it a bit tricky. Even when validation is supported perfectly, there may be times when custom validations are needed and a more manual, Vue-based solution may be more appropriate. Let's begin with a simple example.
+
+For my first example, let's create something as simple as possible. Given a form of three fields, make two required. Let's look at the HTML first:
+
+``` html
+<form id="app" @submit="checkForm" action="/something" method="post">
+  
+  <p v-if="errors.length">
+    <b>Please correct the following error(s):</b>
+    <ul>
+      <li v-for="error in errors">{{ error }}</li>
+    </ul>
+  </p>
+  
+  <p>
+  <label for="name">Name<label>

```

Comment:
Please indent the labels and inputs within the `p` tags, thanks!

Reply:
FYI, after I make this (and the other edits), I'll be sure to update the codepen's too.